app/controllers/catalog_controller.rb

```diff
@@ -423,7 +423,7 @@ def active_sort_fields
 
     config.add_sort_field("recently_added") do |field|
       field.label = "recently added"
-      field.sort = "date_created_dtsi desc"
+      field.sort = "date_published_dtsi desc, date_created_dtsi desc"
```

Comment:
don't we need to pay attention to "nulls last" here again (and test in tests maybe? Not sure if there are already suitable analagous tests somewhere).

Reply:
Nulls are in fact sorted last in practice; a test couldn't hurt.

Reply:
Oh right, this is Solr not postgres!

If there's already similar tests and a covenient place to put the tests, let's do it. Otherwise, not worth too much trouble, if you tested manually.

Reply:
Added tests. (They fail!)